<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/img/ICON</normaloff>:/img/ICON</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>串  口</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>105</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>77</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>133</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>161</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>流  控</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbSerialPort">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>135</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comParity">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>104</y>
       <width>135</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comBaudRate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>48</y>
       <width>135</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comDataBits">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>76</y>
       <width>135</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comFlowControl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>135</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RTS/CTS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XON/XOFF</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comStopBits">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>132</y>
       <width>135</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>接收设置</string>
    </property>
    <widget class="QRadioButton" name="radioRxHex">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioRxAscii">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>ASCII</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxWordWrap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>自动换行</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxShowTime">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>77</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示时间</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxShowSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>77</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>325</y>
      <width>221</width>
      <height>160</height>
     </rect>
    </property>
    <property name="title">
     <string>发送设置</string>
    </property>
    <widget class="QRadioButton" name="radioTxHex">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>33</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>115</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRepeat">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>116</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxCRC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CRC</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxCRCExchange">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>高低字节交换</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbCRCType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxRepeat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>重复发送</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioTxAscii">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ASCII</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textShowRx">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>17</y>
      <width>680</width>
      <height>340</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTx">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>373</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>835</x>
      <y>373</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>发送</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#btnSend{
	
	border-radius: 10px;
	border:0px;
}

QToolButton#btnSend:hover{
    background-color: rgba(117,208,249,100);
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="MainWindow.qrc">
      <normaloff>:/img/发送</normaloff>:/img/发送</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbSendHistory">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>463</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="start"/>
   <addaction name="pause"/>
   <addaction name="stop"/>
   <addaction name="clean"/>
   <addaction name="CRC"/>
   <addaction name="CMD"/>
  </widget>
  <action name="start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/img/启动</normaloff>:/img/启动</iconset>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </action>
  <action name="pause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/img/暂停</normaloff>:/img/暂停</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="stop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/img/停止</normaloff>:/img/停止</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
  </action>
  <action name="clean">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/img/清理</normaloff>:/img/清理</iconset>
   </property>
   <property name="text">
    <string>清理</string>
   </property>
  </action>
  <action name="CRC">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/img/CRC</normaloff>:/img/CRC</iconset>
   </property>
   <property name="text">
    <string>CRC校验</string>
   </property>
   <property name="toolTip">
    <string>CRC校验</string>
   </property>
  </action>
  <action name="CMD">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/img/CMD</normaloff>:/img/CMD</iconset>
   </property>
   <property name="text">
    <string>快捷指令</string>
   </property>
   <property name="toolTip">
    <string>快捷指令</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
